vmdb/app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -51,7 +51,16 @@ def save_container_groups_inventory(ems, hashes, target = nil)
                 []
               end
 
-    save_inventory_multi(:container_groups, ems, hashes, deletes, [:ems_ref], [:container_definitions, :containers, :labels])
+    hashes.each do |h|
```

Comment:
@blomquisg as I said I have the feeling that I am not doing the things by the book here. Basically on the parser side the only way I have to identify a node is its name (and the fact that I know it's unique in the ems). So here I have to search for it using the name and the ems id. I have no ids etc on the parser side.

Do you have a more elegant (or better performing) solution?